src/pages/help-centre/_help-articles.js

```diff
@@ -2,95 +2,391 @@ import React from 'react'
+                    label: 'clear-transaction-log',
+                },
+                {
+                    title: <Localize translate_text="How do I control my losses with DBot?" />,
+                    category: 'DBot',
+                    sub_category: <Localize translate_text="How to trade" />,
+                    label: 'control-loss',
+                },
+                {
+                    title: (
+                        <Localize translate_text="Where can I see the status of my trades in DBot?" />
+                    ),
+                    category: 'DBot',
+                    sub_category: <Localize translate_text="Logs" />,
+                    label: 'status-of-trades',
+                },
+                {
+                    title: <Localize translate_text="How do I view the chart in DBot?" />,
+                    category: 'DBot',
+                    sub_category: <Localize translate_text="Chart" />,
+                    label: 'view-chart',
+                },
+                {
+                    title: <Localize translate_text="What is the martingale strategy?" />,
+                    category: 'DBot',
+                    sub_category: <Localize translate_text="Strategy" />,
+                    label: 'martingale-strategy',
+                },
+                {
+                    title: <Localize translate_text="What is the d’Alembert strategy?" />,
+                    category: 'DBot',
+                    sub_category: <Localize translate_text="Strategy" />,
+                    label: 'dalembert-strategy',
+                },
+                {
+                    title: <Localize translate_text="What is the Oscar's Grind strategy?" />,
+                    category: 'DBot',
+                    sub_category: <Localize translate_text="Strategy" />,
+                    label: 'oscars-grind-strategy',
+                },
+            ],
+        },
+    ],
+    [
+        {
+            category: <Localize translate_text="Security" />,
+            articles: [
+                {
+                    title: <Localize translate_text="Do I need to verify my Deriv account?" />,
+                    category: 'Security',
+                    sub_category: <Localize translate_text="Verification" />,
+                    label: 'verify-account',
+                },
+                {
+                    title: <Localize translate_text="How long does verification take?" />,
+                    category: 'Security',
+                    sub_category: <Localize translate_text="Verification" />,
+                    label: 'verification-duration',
+                },
+                {
+                    title: <Localize translate_text="Why were my documents declined?" />,
+                    category: 'Security',
+                    sub_category: <Localize translate_text="Verification" />,
+                    label: 'documents-declined',
+                },
+            ],
+        },
+    ],
```

Comment:
Is there any necessity to use an `array of array` at the first level? The search is not working anymore and I think it's due to the `match-sorter` package which is used in the code; Apparently, it only can handle a flat array. I think we have to either change the function of `getAllArticles()` or `_help-articles` array. What's your idea?

Reply:
I did it so that it could show three items in one row. I think it makes it confusing to use this structure to pass to DOM, please update it however you see fit. If we create components I think it's more readable than an array, but you know the codebase better than me.